admin/templates/configuration/etc/log4cxx.czar.properties

```diff
@@ -0,0 +1,27 @@
+#
+# Configuration file for log4cxx
+# used by Qserv-czar
+#
+
+log4j.rootLogger=DEBUG, FILE
+
+# Use this appender to interlace xrootd debug messages with Qserv-czar standard
+# logs. stdout/stderr are redirected to {{QSERV_LOG_DIR}}/qserv-czar-console.log.
+# This workaround is required because Qserv-czar can only outputs xrootd debug
+# message on stderr
+log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
```

Comment:
No it isn't used, if you want to activate it you have to write 

log4j.rootLogger=DEBUG, FILE, CONSOLE

Is it fine to keep it as an example?